src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.cs

```diff
@@ -24,6 +24,7 @@ public static partial class PlatformDetection
         public static bool IsFullFramework => RuntimeInformation.FrameworkDescription.StartsWith(".NET Framework", StringComparison.OrdinalIgnoreCase);
         public static bool IsNetNative => RuntimeInformation.FrameworkDescription.StartsWith(".NET Native", StringComparison.OrdinalIgnoreCase);
         public static bool IsNetCore => RuntimeInformation.FrameworkDescription.StartsWith(".NET Core", StringComparison.OrdinalIgnoreCase);
+        public static bool IsMonoRuntime => Type.GetType("Mono.RuntimeStructs") != null;
```

Comment:
When I add a new property here my corefx code don't see it, do I need any additional work to update CoreFx.Private.TestUtilities? cc: @ViktorHofer

Reply:
You need to add it to the ref for CoreFx.Private.TestUtilities as well, and build both the ref and the src.